assessments/projects/longhorn/self-assessment.md

```diff
@@ -0,0 +1,436 @@
+
+Vulnerability Response Process
+* Responsible Team: The Longhorn security team is tasked with responding to security reports. This team includes maintainers who have a deep understanding of the codebase and security practices. The vulnerabilities can then be reported to [longhorn-security](mailto:longhorn-security@suse.com).
+* Reporting Process: Reporters are asked to include detailed information about the issue, steps to reproduce, and any other relevant data. If having any vulnerabilities found, Longhorn askes it to be reported to [longhorn-security](longhorn-security).
+* Response Actions: The team assesses the report for validity, determines the severity, develops a fix, and then coordinates a release timeline for the patch. They also communicate with the reporter throughout the process, providing updates and seeking additional information if necessary.
+
+Incident Response
+* Triage: Security incidents are prioritised based on severity, impact, and complexity. The team rapidly assesses the scope to understand the breadth of the impact.
+* Confirmation: The team confirms the incident and identifies affected systems and data.
+* Notification: Stakeholders, including users and potentially affected parties, are informed about the incident as appropriate. Communication is carefully managed to ensure transparency while avoiding unnecessary alarm.
+* Patching: Development of a security patch is expedited, and upon completion, the patch is made available. Users are notified of the update and provided with instructions for applying the patch.
+* Post-Incident Analysis: After resolving the incident, the team conducts a post-mortem to understand the root cause, improve future response efforts, and enhance security measures to prevent similar incidents.
+
+
+## Appendix
+Known Issues Over Time
+* Longhorn tracks all issues publicly on their [GitHub issues page](https://github.com/longhorn/longhorn/issues). Any security vulnerabilities discovered in the past are documented there along with their resolutions.
+* The project maintains a [security advisory](https://github.com/longhorn/longhorn/security/advisories) page where they publish details about the security vulnerabilities found.
+* Statistics on vulnerabilities found and fixed are not explicitly listed unless provided on the project’s repository or documentation. If available, data about the effectiveness of code reviews and automated testing in catching issues would be included.
+* For a detailed history of known issues, visit: [Known Issues Over Time](https://longhorn.io/blog/cve-2021-36779-36780/).
+
+CII Best Practices
+* Longhorn is actively working towards meeting the CII Best Practices criteria.
+* Longhorn’s commitment to security can be observed in their adherence to these practices, such as using automated testing, code review, and maintaining a public version control repository. More details can be found here: [CII Best Practices](https://longhorn.io/docs/1.5.3/best-practices/)
```

Comment:
The link provided talks about the best practices to use Longhorn, What needs to be documented here is the best practices followed by Long horn project in updating, maintaining and securing the project. If Longhorn doesn't have CII badge for the project or any corresponding entry, add that as an action item for the project.

Reply:
Thank you, Mr. Pranava. We have updated the action item to include acquiring a CII badge as well. Thank you for pointing this out